Knysh/Dijkstra/dj.C

```diff
@@ -0,0 +1,58 @@
+// spiszhenno, no zato razobralsa
+#include <iostream>
+#include <vector>
+#define INT_MAX 100000
```

Comment:
const are to be made with `const ...`, not define.
INT_MAX is reserved. Use `MAX_DISTANCE` or something similar.